src/Host/Broker/Impl/About/AboutController.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Microsoft.R.Host.Broker.About {
     [Authorize(Policy = Policies.RUser)]
-    [Route("/about")]
```

Comment:
If you just want to distinguish different things, add subpoints under `/about`  (and maybe rename it to `/info`). So we can have `/info/host`, `/info/load` etc.

Although I'd make it `/status/load` etc instead. `/about` or `/info` should be for static stuff that doesn't change for a particular instance of broker, like version numbers or install locations.